cmd/minishift/cmd/hostfolder/add.go

```diff
@@ -93,13 +91,15 @@ func addHostFolder(cmd *cobra.Command, args []string) {
 		// Windows-only (CIFS), all instances
 		name = "Users"
 	} else {
-		if interactive && len(args) < 1 {
-			atexit.ExitWithMessage(1, interactiveModeUsage)
-		}
-		if len(args) < 1 {
+		if len(args) < 1 && !interactive {
 			atexit.ExitWithMessage(1, usage)
 		}
-		name = args[0]
+		if interactive {
+			name = util.ReadInputFromStdin("Name")
+			shareType = util.ReadInputFromStdin("Type")
```

Comment:
Better to put `Type [sshfs|cifs]` instead just `Type` so user don't need to check back to doc/reference what type can be.

Reply:
Done.. but, also included the short forms as suggested in https://github.com/minishift/minishift/pull/2324#issuecomment-383469926

`Type [ (s)shfs | (c)ifs ]`